media/css/firefox/accounts-2019.scss

```diff
@@ -200,7 +200,7 @@
     height: 50px;
 
     .t-product-firefox & {
-        @include at2x('/media/img/logos/firefox/quantum/logo-word-hor-stack-md.png', 144px, 50px);
+        @include at2x('/media/protocol/img/logos/firefox/browser/logo-word-hor-stack-md.png', 144px, 50px);
```

Comment:
Note there's another logo here that just merged https://github.com/mozilla/bedrock/blob/master/media/css/exp/firefox/accounts-2019.scss#L203

Reply:
Thanks, I'll update.